brainatlas_api/structures/structure_tree.py

```diff
@@ -389,3 +396,93 @@ def path_to_list(path):
             return list(path)
 
         return [int(stid) for stid in path.split("/") if stid != ""]
+
+    def print_structures(self, to_file=False, save_filepath=None):
```

Comment:
I think this is a convenient function and I see the point of having it here; I am a splitting maniac and I would keep it as a separate function then used here, put it's a matter of taste - as long as this class does not get too long

Reply:
> I am a splitting maniac 

You really are :P

I prefer keeping the number of files / classes low at the cost of slightly longer classes. Especially for stuff like this which is behind the scenes and we won't have to edit it often if at all. 

I guess we can find a compromise? :)

Reply:
I would agree on a maximum number of lines per class 😄  500?